sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/README.md

```diff
@@ -1,74 +1,78 @@
+3. Under **webapp** application, select **Certificates & secrets** -> **new client secret**, expires select **Never**, click the **add** button.(Remember to save the secrets here and use them later)![Creat secrets](docs/image-creat-secrets-app.png "Creat secrets")
+4. Under **webapp** application, select **Authentication** -> **Add a platform**, select **web** platform, redirect urls set to `http://localhost:8080/login/oauth2/code/azure`, check the Access Token and ID Token checkboxes, click **configure** button.![Add a platfform](docs/image-add-a-platfform.png "Add a platfform")
+5. Under **webapp** application, select **Authentication** -> **Add URI**, you need to add redirect URIs of `http://localhost:8080/login/oauth2/code/arm`. ![App add url](docs/image-app-add-url.png "App add url")
+6. Under **webapp** application, select **API permissions** -> **Add a permission**, select **Microsoft Graph**. Next, search **Calendar.Read** via select Permissions, check the check box,click **add permissions** button.![Api permission](docs/image-api-permissions.png "Api permission")
+7. Similarly, add **Directory.AccessAsUser.All** permission in **Microsoft Graph** and add **ActivityFeed.Read** permission in **Office 365 Management APIs**.![Add Directory.AccessAsUser.All and ActivityFeed.Read permission](docs/image-add-accessuser-and-activityfeed-permission.png "Add Directory.AccessAsUser.All and ActivityFeed.Read permission")
+8. Continue to select **API permissions** -> **Add a permission** -> **My APIS**, select **arm** tab, add **Files.read** permission,click **Add permissions** button.![Webapi permission](docs/image-webapi-permission.png "Webapi permission")
+9. click **Grant admin consent for...**.![Grant permission](docs/image-granted-permission.png "Grant permission")
+10. Finally.![Final](docs/image-final.png "Final")
+
+See [Register app or web api], [Grant scoped permission] for more information about the Web APP.
 
 ### Configure groups for sign in user
-
-In order to try the authorization action with this sample with minimum effort, [configure the user and groups in Azure Active Directory](https://docs.microsoft.com/azure/active-directory/active-directory-groups-create-azure-portal), configure the user with `group1`. 
-
+In order to try the authorization action with this sample with minimum effort, [configure the user and groups in Azure Active Directory], configure the user with `group1`.
 
 ## Examples
-
 ### Configure application.yml
-
 ```yaml
 azure:
   activedirectory:
-    client-id: <client-id>
-    client-secret: <client-secret>
+    authorization:
+      arm:
+        on-demand: true
+        scopes: <your-arm-scope-url>
+      graph:
+        scopes: https://graph.microsoft.com/Calendars.Read
+      office:
+        scopes: https://manage.office.com/ActivityFeed.Read
```

Comment:
Please use the same scope with aad integration test module:
![image](https://user-images.githubusercontent.com/13167207/103070293-3056c480-45fc-11eb-93a9-abe4726b02a8.png)

Reply:
@ZhuXiaoBing-cn , we need to update the scopes here.